crates/test-utils/src/fixtures.rs

```diff
@@ -5,16 +5,16 @@ use crate::execution::contract::KakarotEvmContract;
 use crate::sequencer::Katana;
 
 #[fixture]
-#[awt]
```

Comment:
why did you remove this?

Reply:
i thought compiler is getting confused in some macro, so i was trying to remove `#[awt]` which manually awaits the future and did it manually, but that still didnt fix it

tldr; the issue persists with and without `#[awt]` macro

Reply:
https://github.com/kkrt-labs/kakarot-rpc/pull/656#discussion_r1400738718